tissue-clicker/src/App.js

```diff
@@ -1,21 +1,30 @@
-import { useState, useEffect } from 'react';
+import { BrowserRouter as Router, Route, Routes, Link } from 'react-router-dom';
 import './App.css';
+import Home from './Home';
+import Rollers from './Rollers';
+import Decor from './Decor';
+import About from './About';
 
 function App() {
-  const [clickCount, setClickCount] = useState(0);
-
-  function handleClick() {
-    setClickCount(clickCount + 1);
-  }
-
   return (
-    <div className="App" onClick={handleClick}>
-      <h1>Tissue Clicker Game</h1>
-      <p>Click Count: {clickCount}</p>
-      <div className="tissue-roll">
-        <p>🧻</p>
+    <Router>
+      <div className="App">
+        <nav>
+          <ul>
+            <li><Link to="/">Home</Link></li>
+            <li><Link to="/rollers">Rollers</Link></li>
+            <li><Link to="/decor">Decor</Link></li>
+            <li><Link to="/about">About</Link></li>
+          </ul>
+        </nav>
+        <Routes>
```

Comment:
Originally had this as a <Switch> tag, but seems this version needs <Routes> instead